### PR TITLE
Sync to sweetalert v1.1.3

### PR DESCRIPTION
**Pull request by alekitto**
Should be complete, but needs testing

I've not sync'ed examples with input, loaders, etc: i will add them in a separate commit as soon as possible.

As this is a merge where files have been splitted, moved, etc, probably i've broken something, so feedbacks on non-working features are welcome

### CODE REVIEW

**Comment by lipis** [comment]
You are doing God's work my friend..!

**Comment by lipis** [comment]
I'll keep it for couple of days if anyone is going to test it/comment/etc.. and whenever you want.. we can merge it.

**Comment by alekitto** [comment]
👍 
Meanwhile i'll do some test in the weekend, i'll write here (or commit) if i find something to fix

It was just a too exciting challenge for refusing...

**Comment by lipis** [comment]
Feel free to continue committing here.. this project is way behind to wait :)

**Comment by lipis** [comment]
How is it going here?

File: README.md [hunk 1]
```diff
@@ -1,10 +1,3 @@
-> **Attention**! This project is currently behind the original
->[SweetAlert](https://github.com/t4t5/sweetalert) and **not maintained**. If anyone is brave enough to
-> to successfully [sync it](https://github.com/lipis/bootstrap-sweetalert/compare/master...t4t5:master),
-> the community would be entirely greatufl.
-
--------------------------------------------------------------------------------
-
```
Comment (lipis) [inline README.md]: 💯

File: README.md [hunk 2]
```diff
@@ -36,3 +29,10 @@ match your theme perfectly.
 $ npm install
 $ grunt
 ```
+
+### Run tests
+```shell
+$ npm install
+$ bower install
```
Comment (lipis) [inline README.md]: add this line to the `package.json` like here: https://github.com/gae-init/gae-init/blob/master/package.json#L10

Comment (alekitto) [inline README.md]: 👍

**Comment by alekitto** [comment]
I think it's done. Input errors classes are fixed (show the glyphicon exclamation mark with error message), and a couple of warnings are removed.

Switching to sass and gulp should be considered (could help maintain this project in sync), but out of the scope of this PR.

Ok for me to merge. I'll open some issues for other ideas to discuss them further.

**Comment by alekitto** [comment]
I merged the travis configuration file too, so could (and should) be enabled after the merge

**Comment by lipis** [comment]
thanks a bunch!!!

**Comment by lipis** [comment]
welcome :)

**Comment by alekitto** [comment]
Thanks :)

I think some old issues can be closed now...

**Comment by lipis** [comment]
feel free! you have the power..

**Comment by lipis** [comment]
are you on Twitter?

**Comment by alekitto** [comment]
Now i am: https://twitter.com/alekitto

**Pull request merged**